fontc_crater/src/ci/html.rs

```diff
@@ -172,6 +153,27 @@ fn make_table_body(runs: &[RunSummary]) -> Markup {
     }
 }
 
+/// for values that change between runs, make the little decoration span that says like,
+/// "+0.333" or "-5"
+fn make_delta_decoration<T: PartialOrd + Copy + Sub<Output = T> + Display + Default>(
+    current: T,
+    prev: Option<T>,
+    more_is_better: bool,
+) -> Markup {
+    let delta = prev.map(|prev| current - prev).unwrap_or(current);
+    let diff = format!("{delta:+.3}");
+    match prev.and_then(|prev| current.partial_cmp(&prev)) {
+        Some(std::cmp::Ordering::Equal) => Default::default(),
+        // don't write +0
+        None if current == T::default() => Default::default(),
+        None | Some(std::cmp::Ordering::Greater) if more_is_better => {
+            html! { span.better { (diff) } }
+        }
+        Some(std::cmp::Ordering::Less) if !more_is_better => html! { span.better { (diff) } },
```

Comment:
shouldn't this be |'d onto the case above?

Reply:
there can only be a single `if` clause on a match arm, and it applies after the match (it is not associated with a specific match item, see https://doc.rust-lang.org/1.80.1/reference/expressions/match-expr.html#match-expressions)

Reply:
TIL. Or re-learned I suppose since I'm sure I did read that at some point.